src/styles/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+  transition-property: background-color, color, border;
```

Comment:
The `transition` shorthand property could be used here instead of specifying `transition-duration` and `transition-property` separately. This would make the code cleaner and more concise. For example: `transition: background-color 300ms, color 300ms, border 300ms;`.